gammapy/maps/wcs.py

```diff
@@ -741,31 +741,63 @@ def separation(self, center):
         coord = self.to_image().get_coord()
         return center.separation(coord.skycoord)
 
-    def get_region_mask_array(self, region):
-        """Return mask of pixels inside region in the the form of boolean array
+    def region_mask(self, regions, inside=False):
+        """Create a mask from a given list of regions
+
+        TODO: implement list of region for each axis
 
         Parameters
         ----------
-        region : `~regions.PixelRegion` or `~regions.SkyRegion` object
-            A region on the sky could be defined in pixel or sky coordinates.
+        regions : list of  `~regions.PixelRegion` or `~regions.SkyRegion` objects
+            A list of regions on the sky (defined in pixel or sky coordinates).
+        inside : bool
+          Output array is set to True inside the input region if inside is set to True and False outside and conversely.
 
         Returns
         -------
-        mask_array : `~numpy.ndarray` of booleans
-            the array of booleans
+        mask_map : `~numpy.ndarray` of boolean type
+            the mask map
+
+        Example
+        -------
+        Example building a mask and storing it in a `~gammapy.maps.WcsNDMap' ::
```

Comment:
I think you need an empty line here, or the docs example won't be properly formatted in the HTML output, no?

Reply:
done